examples/rotation_bin/main.rs

```diff
@@ -0,0 +1,71 @@
+mod rotation_bin;
+
+use rotation_bin::{Rotation, RotationBin};
+
+use gtk::glib;
+use gtk::glib::clone;
+use gtk::prelude::*;
+
+fn main() {
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.rotation_bin"),
+        Default::default(),
+    );
+
+    application.connect_activate(build_ui);
+    application.run();
+}
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(application);
+
+    let vbox = gtk::Box::new(gtk::Orientation::Vertical, 6);
+    let rotate_bin = RotationBin::new();
+    let img = gtk::Image::builder()
+        .pixel_size(128)
+        .icon_name("audio-x-generic")
+        .build();
+    rotate_bin.set_child(Some(img.upcast_ref()));
+
+    let grid = gtk::Grid::builder()
+        .margin_start(6)
+        .margin_end(6)
+        .margin_top(6)
+        .margin_bottom(6)
+        .halign(gtk::Align::Center)
+        .valign(gtk::Align::Center)
+        .row_spacing(6)
+        .column_spacing(6)
+        .build();
+
+    let clockwise = gtk::Button::with_label("Rotate clockwise");
+    let counter_clockwise = gtk::Button::with_label("Rotate counter clockwise");
+
+    clockwise.connect_clicked(clone!(@weak rotate_bin => move |_| rotate_bin.rotate_clockwise()));
+    counter_clockwise.connect_clicked(clone!(@weak rotate_bin => move |_| rotate_bin.rotate_counter_clockwise()));
```

Comment:
any preference on how to format these? Currently they are a bit long ~105 chars.

Reply:
```suggestion
    counter_clockwise.connect_clicked(clone!(@weak rotate_bin => move |_| {
        rotate_bin.rotate_counter_clockwise();
    }));
```
would do something like this

Reply:
changed